### PR TITLE
Improve logging

### PR DESCRIPTION
**Pull request by fpob**
### Fixes

- [EXDRHUB-1088](https://issues.redhat.com/browse/EXDRHUB-1088)

### Checklist

- [x] Run `tox`, no tests failed

### CODE REVIEW

**Comment by freedgit** [comment]
Since this has no changes to logging.*, I assume it still goes to stdout, right? Bc in that case we can just redirect everything to a centralized logger using fluent-bit, plain rsyslog via docker config etc

**Comment by fpob** [comment]
@freedgit It depends how you have configured logging. By default (LOG_LEVEL env var), the logging output is console, but you can configure whatever you want in the config file (LOG_CONFIG env var) https://github.com/resource-hub-dev/rhub-api/blob/master/config/logging.example.yml

File: src/rhub/api/tower.py [hunk 1]
```diff
@@ -364,6 +395,11 @@ def relaunch_job(job_id, user):
         db.session.add(relaunched_job)
         db.session.commit()
 
+        logger.info(
+            f'Re-aunching tower job {tower_job_data["id"]}',
```
Comment (brhatiga) [inline src/rhub/api/tower.py:399]: Re-launching

**Review by brhatiga — CHANGES_REQUESTED** [review]
LGTM, just a small spelling issue

**Review by brhatiga — APPROVED** [review]

**Pull request merged**